templates/apps/sumo-aws-apps/common-org/common-org-management.yaml

```diff
@@ -952,6 +868,158 @@ Resources:
+      EventPattern:
+        source:
+          - aws.organizations
+        detail-type:
+          - AWS API Call via CloudTrail
+        detail:
+          eventSource:
+            - organizations.amazonaws.com
+          eventName:
+            - AcceptHandshake
+            - CreateAccountResult
+            - RemoveAccountFromOrganization
+      State: ENABLED
+      Targets:
+        - Arn: !GetAtt rSumoLogicLambdaHelper.Arn
+          Id: !Sub CommonOrgManagement-SumoLogicLambdaHelper-${ParentStackName}
+
+  PermissionForOrganizationsRuleToInvokeLambda:
+    Condition: create_config_recorder_role  
+    Type: AWS::Lambda::Permission
+    Properties:
+      FunctionName: !GetAtt rSumoLogicLambdaHelper.Arn
+      Action: lambda:InvokeFunction
+      Principal: events.amazonaws.com
+      SourceArn: !GetAtt OrganizationsRule.Arn
+
+  CrossRegionEventRuleRole:
+    Type: AWS::IAM::Role
+    Condition: NotGlobalRegionUsEast1
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W28
+            reason: Specific role name provided
+    Properties:
+      RoleName: !Sub "config-recorder-start-${ParentStackName}"
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Action: sts:AssumeRole
+            Principal:
+              Service:
+                - events.amazonaws.com
+      Policies:
+        - PolicyName: config-events
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: events:PutEvents
+                Resource: !Sub arn:${AWS::Partition}:events:${AWS::Region}:${AWS::AccountId}:event-bus/default
+
+  ConfigRecorderRoleStackset:
+    Condition: create_config_recorder_role
+    Type: AWS::CloudFormation::StackSet
+    Properties:
+      Tags:
+        - Key: "version"
+          Value: !Ref QSVersion
+      StackSetName: !Sub 'CommonOrgConfigRecorderRole-${ParentStackName}'
+      AutoDeployment:
+        Enabled: true
+        RetainStacksOnAccountRemoval: true
+      Parameters:
+        - ParameterKey: ConfigRecorderRoleName
+          ParameterValue: !Sub "ConfigRecorderRole-${ParentStackName}"
+        - ParameterKey: BucketName
+          ParameterValue: !If [create_s3_bucket, !GetAtt rDeliveryS3BucketArn.ARN, !Ref CloudTrailExistsS3BucketName]
+        - ParameterKey: ManagementAccountId
+          ParameterValue: !Ref AWS::AccountId
+        - ParameterKey: ConfigAssumeRoleName
+          ParameterValue: !Sub "Config-Recorder-${ParentStackName}"
+        - ParameterKey: OrgLambdaRoleName
+          ParameterValue: !Select [1, !Split ["/", !GetAtt rSumoLogicHelperRole.Arn]]
+      PermissionModel: SERVICE_MANAGED
+      Capabilities:
+        - 'CAPABILITY_NAMED_IAM'
+        - 'CAPABILITY_IAM'
+        - 'CAPABILITY_AUTO_EXPAND'
+      StackInstancesGroup:
+        - Regions:
+            - !Ref AWS::Region
```

Comment:
shouldn't this be config regions (where config needs to be enabled) ?

Reply:
no need.  Because Roles can be deployed in any region